### DISCUSSION TITLE
ip/webhook showing method not allowed

### DISCUSSION THREAD

**Discussion by pooka99** [FabCx/TradingView-Webhook-Bot#123] [Q&A]
Running centos off Hetzner no firewall active, did everything and python main.py not throwing any errors. server listening on 0.0.0.0  port 80.  confirmed trigger is firing on TV but discord not seeing it.  Please advice.

![iScreen Shoter - Google Chrome - 230626220156](https://github.com/fabston/TradingView-Webhook-Bot/assets/54335169/8223bf16-768c-4004-94d9-828a862a60bc)

**Comment by xgaben07** [comment]
same rightnow

**Comment by GuillaumeMoscou** [comment]
same for me 

![image](https://github.com/fabston/TradingView-Webhook-Bot/assets/98876771/fa78b36e-198a-4171-a7df-bf562df9542d)

**Comment by nDman** [comment]
same for me :(
